tests/queries/0_stateless/02500_file_url_function.sql

```diff
@@ -0,0 +1,18 @@
+-- Проверка, что table function URL работает с file:// схемой
+DROP TABLE IF EXISTS url_file_test;
+
+CREATE TABLE url_file_test
+(
+    id UInt32,
+    name String
+)
+ENGINE = Memory;
+
+SYSTEM CREATE FILE '/tmp/test.csv' '1,Alexey\n2,Yuliia';
+
+SELECT *
+FROM url('file:///tmp/test.csv', 'CSV', 'id UInt32, name String')
```

Comment:
This will fail since the file is not inside `user_files` directory, and we need to leave this test case to ensure that this is prohibited, i.e.

```
select * from url('file:///tmp/test.csv', 'csv'); -- { serverError DATABASE_ACCESS_DENIED }
```

And also add a test inside `user_files`, for this you need `.sh` test and use `USER_FILES_PATH` (see at i.e. `tests/queries/0_stateless/02892_input_csv_cr_end_count_many_rows.sh`)